scripts/systemd/nut-server.service.in

```diff
@@ -57,6 +57,7 @@ SyslogIdentifier=%N
 # Note: foreground mode "-F" by default skips writing a PID file (and
 # needs default Type=simple); we can use "-FF" here to create the file
 # anyway, so that old "upsd -c reload" works rather than systemd action:
+ExecCondition=@SBINDIR@/sh -c '[ "$MODE" != "netclient" ]'
```

Comment:
`@SBINDIR@` is for NUT's system binaries; this is not necessarily where OS binaries co-exist.
I think Unix standards dictate that a `/sbin/sh` (for system scripts) exists, may be or not be same as `/bin/sh` (default for interactive users) - so please rather refer to one of those.

Reply:
You are right, I've check 
**debian:**
bin -> usr/bin
bin -> usr/sbin
/usr/sbin/upsd
/usr/bin/sh

Arch:
bin -> usr/bin/
sbin -> usr/bin/
/usr/bin/upsd
/usr/bin/sh

So it's the same PATH on Arch but not debian. It's in `/bin/sh` / `/usr/bin/sh` on both cases.